assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -42,5 +42,18 @@ export const tests: Tests = {
     independent: true,
     seed: 7,
   },
+
+  upsellRecurringContributions: {
+    variants: ['control', 'benefitsOfBoth', 'shorterControl'],
+    audiences: {
+      US: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    isActive: true,
+    independent: true,
```

Comment:
Just to double check, we will not use the `mvt_id` to assign a user to a Variant, is this what we want?

Reply:
yeah think that's fine - @jranks123 or @joelochlann - could you confirm?

Reply:
It is only in very rare cases that we would ever want to use the mvt id, as this can lead to skewed test results if there is a test running on the same audience segment in frontend, so this should be independent.